apps/controllerx/devices/aqara.py

```diff
@@ -26,3 +26,29 @@ def get_z2m_actions_mapping(self):
             "right_double": Light.CLICK_BRIGHTNESS_UP,
             "right_long": Light.CLICK_BRIGHTNESS_DOWN,
         }
+
+
+class MFKZQ01LMController(LightController):
+    """
+    This controller allows movement actions for Xiaomi Aqara Smart Cube as
+    shake, wakeup, fall, slide, flip180 or 90 and rotate_left or right. 
+    Then from the apps.yaml the needed actions can be included and create
+    different instances for different lights.
+    """
+
+    # Different states reported from the controller:
+    # shake, wakeup, fall, tap, slide, flip180
+    # flip90, rotate_left and rotate_right
+
+    def get_z2m_actions_mapping(self):
+        return {
+            "shake": Light.TOGGLE,
```

Comment:
In the custom-controller branch, I have added a new predefined action that lets you set the light to minimum brightness or color temperature, so I will replace this to minimum brightness. And use tap as `toggle` which feels more natural. As of nw, you do not need to change anything in this line, I will do these changes with the merge.